.gitignore

```diff
@@ -74,7 +74,7 @@ gradle-app.setting
 *.zip
 *.tar.gz
 *.rar
-
+configuration.toml
```

Comment:
Remove this configuration.toml is already in there

Reply:
oh sorry it was me configurating my server

Reply:
fixed